hs_collection_resource/templates/pages/collectionresource.html

```diff
@@ -0,0 +1,553 @@
+                            <span title='Download All Content as Zipped BagIt Archive' class="button-label">Download Collection Bag</span>
+                        </span>
+                        <a target="_blank" href="http://en.wikipedia.org/wiki/BagIt">Learn more about the Bagit archive format</a>
+                    </a>
+                {% endfor %}
+            </div>
+        {% endblock %}
+    </div>
+
+    {#    modal div: warning message  #}
+    <div class="modal fade" id="save-collection-warning" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                    <h4 class="modal-title" id="save-collection-warning-title">Confirm Changes</h4>
+                </div>
+                <div class="modal-body" id="save-collection-warning-body">
+                    You have NO PERMISSION over the resource you are about to remove from this collection. Once confirmed, you CAN NOT add it back.
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-default" data-dismiss="modal" id="save-collection-btn-cancel">
+                        Cancel
+                    </button>
+                    <button type="button" class="btn btn-danger" data-dismiss="modal" id="save-collection-btn-ok">
+                        Confirm
+                    </button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    {#    modal div: collection candidate table #}
+    <div class="modal fade" id="collection-candidate" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+        <div class="modal-dialog" style="width:65%">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                    <h4 class="modal-title" id="save-collection-warning-title">My Resources Pool</h4>
+                </div>
+                <div class="modal-body" id="save-collection-warning-body">
+                    <table id="collection-table-candidate" class="table-hover table-striped" width="100%">
+                        <thead>
+                        <tr>
+                            <th>Add</th>
+                            <th>Title</th>
+                            <th>Type</th>
+                            <th>Owners</th>
+                            <th>Sharing status</th>
+                            <th>My Permission</th>
+                            <th>Remove</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        {% if collection_candidate %}
```

Comment:
@zhiyuli I think this {% if collection_candidate %} is not needed since the following {% for ...%} block will only be executed when collection_candidate is not empty. I believe there is another place in your template with the same logic, where the extra {% if ... %} check can be removed.